src/relay/index.ts

```diff
@@ -0,0 +1,40 @@
+import 'isomorphic-fetch';
+import {StandardRelayerApi, SignedOrder} from '../types';
+
+export class Relay {
+    private url: string;
+    private version: number;
+    constructor(url: string) {
```

Comment:
Let's already add an options obj to the constructor with the `version` as one of it's optional params.